package.json

```diff
@@ -42,31 +46,43 @@
   },
   "homepage": "https://github.com/livepeer/protocol#readme",
   "devDependencies": {
-    "@nomiclabs/hardhat-ethers": "^2.0.1",
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "@nomiclabs/hardhat-etherscan": "^2.1.4",
+    "@nomiclabs/hardhat-waffle": "^2.0.1",
```

Comment:
Do the waffle dependencies need to be installed with hardhat? Would it be possible to hold off on adding the dependencies until we're actually making use of the dependencies i.e. in a separate PR that introduces waffle usage?

Reply:
No not a mandatory install, will remove.